packages/compass-telemetry/src/experimentation-provider.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { createContext, useContext, useMemo } from 'react';
+
+interface ExperimentAssignmentData {
+  variant: string | null;
+  isInSample: boolean;
+}
+
+interface ExperimentData {
+  assignmentDate: Date;
```

Comment:
Are these types supposed to identically match the types in the [experiment sdk](https://github.com/10gen/mdb-experiment-js-sdk/blob/a01a8e716ba27f514806abce136ebe374852d142/src/types.ts#L176)?
Some of these values are typed slightly differently. For example `assignmentDate` is typed as a `string`.

Reply:
`id` is also nullable in that `types.ts` file too

Reply:
Alternatively, maybe the `mdb-experiment-js-sdk` can be imported into compass's deps and then the types imported from there to here